lxd/instance/drivers/driver_qemu.go

```diff
@@ -410,7 +410,7 @@ func (d *qemu) getClusterCPUFlags() ([]string, error) {
 
 	for _, node := range nodes {
 		// Attempt to load the cached resources.
-		resourcesPath := internalUtil.CachePath("resources", fmt.Sprintf("%s.yaml", node.Name))
+		resourcesPath := shared.CachePath("resources", fmt.Sprintf("%s.yaml", node.Name))
```

Comment:
this change isnt related to the commit message, probably needs to be included in an earlier commit.